style.css

```diff
@@ -317,3 +344,224 @@ span {
   border: 1px solid #7f8cff;
   border-radius: 8px;
 }
+
+.card-text {
+  padding-left: 3%;
+  padding-right: 3%;
+}
+
+@media (min-width: 768px) {
+  .nav-menu li {
+    display: block;
+    color: #344563;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    font-family: 'Poppins', sans-serif;
+  }
+
+  .heading {
+    padding-left: 260px;
+    padding-right: 260px;
+    padding-top: 40px;
+    background-image: url("images/header_bg.png");
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: 45px 25px !important;
+    background-size: 95%;
+  }
+
+  .social-links {
```

Comment:
- [ ] Please increase the gap to `21px` to follow the design guidelines. 😄 
- [ ] They'll look better if you give them fixed width (`140px`).